src/Angor/Client/Pages/Signatures.razor

```diff
@@ -286,6 +286,16 @@
 
             foreach (var pendingSignature in signaturesRequests.Where(_ => _.AmountToInvest == null))
             {
+                if (string.IsNullOrEmpty(pendingSignature.TransactionHex))
```

Comment:
As we discussed rename Transaction Hex to Encrypted message and filter out nulls in the loop
Also we should perhaps add the decryption to the try catch so the whole flow doesn't fail